lib/hackerone/client/asset.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module HackerOne
+  module Client
+    class Asset
+      include ResourceHelper
+
+      DELEGATES = [
+        :asset_type,
+        :identifier,
+        :description,
+        :coverage,
+        :max_severity,
+        :confidentiality_requirement,
+        :integrity_requirement,
+        :availability_requirement,
+        :created_at,
+        :updated_at,
+        :archived_at,
+        :reference,
+        :state,
+      ]
+
+      delegate *DELEGATES, to: :attributes
+
+      def initialize(organization_id, asset)
```

Comment:
The pattern in other objects of this library is currently `object, additional_parameter`:

```suggestion
      def initialize(asset, organization)
```